### DISCUSSION TITLE
does this support multiple request forwarding mappings?

### DISCUSSION THREAD

**Discussion by siddhsql** [mitre/HTTP-Proxy-Servlet#250] [General]
I want to replace `nginx` with a Java based reverse proxy. With `nginx` I can forward the request to different upstream servers depending on the request path. Can I do this with `http-proxy-servlet` as well?

**Comment by ivanfmartinez** [comment]
You can create a filter to be executed before  and set the parameters for the target based on your needs

https://github.com/mitre/HTTP-Proxy-Servlet/blob/b953eeb0842fcadc6a584014611b7b0782f145f8/src/main/java/org/mitre/dsmiley/httpproxy/ProxyServlet.java#L346

**Reply by dsmiley** [reply]
BTW, really appreciate to see someone other than me answer a question -- thanks Ivan!
